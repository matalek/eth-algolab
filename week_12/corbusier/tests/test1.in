10
20 757 995
884824675
244531555
640336790
771689470
332832395
354472080
427916915
796697395
881336910
346000015
904089785
443874435
249031575
303303590
681801255
442113920
829557770
573209360
209793600
65664020
20 432 998
820622632
520567303
365137645
387740195
722094029
195453751
401080537
806140369
555029690
291356199
223630798
979534918
814318769
965009680
759284335
72204152
792810033
678611640
681306072
202761231
20 959 998
364433112
954059444
402108282
548460182
545747412
253442070
961641790
222082408
280324556
687728644
467917814
692370792
585674456
95478318
680749448
749647794
473288436
11630006
623586034
158494994
20 398 995
805865701
648504917
789308342
965819791
988894972
664726743
25753607
167326298
746116904
664579406
564275861
108151865
681614816
277160
191287734
542150982
896279497
416651421
814595340
42988763
20 850 979
131293561
472662495
690809715
32889826
627750053
369066905
510707658
776254492
397476140
18910934
572087885
626040418
368688687
362722319
742883352
366718720
461393170
812425546
566888699
448370651
20 630 893
378775229
881441760
341882450
15520877
127998513
719839530
606382534
273673352
864147184
553575012
149758160
382211689
82176873
58343538
644411106
203803275
23705949
84245338
729804168
357095631
20 386 893
903834826
407827229
823365912
773213778
494025213
168036551
17657061
196502940
716993785
994638524
969827697
917628788
860398413
839253598
547892759
556790630
462306328
541616926
956170288
476737531
20 199 893
23606379
801799905
807886118
127889247
754275395
493520212
556358703
587435844
135858759
132270020
985795297
970635843
315667083
193102662
676677267
585250502
640146499
48138096
236388994
644333282
20 781 893
965197264
635660618
291448429
930193868
55972100
993738874
591329457
336296067
867460029
365106660
328573897
500213665
178882587
313353985
964990601
466871363
191856604
958261200
985630567
536957290
20 162 973
281015401
950149216
209513980
989071005
574335342
636405230
543519946
592797807
244178306
308706016
267527330
424968393
100540454
803369042
209028844
291552942
483387449
295355762
262192790
129139278
