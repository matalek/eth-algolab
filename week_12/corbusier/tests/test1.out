no
yes
no
yes
yes
yes
no
no
no
yes
